tox.ini

```diff
@@ -17,6 +17,7 @@ passenv =
     aws_access_key_id
     aws_secret_access_key
     mturk_worker_id
+    threads
```

Comment:
Nice - I'd done the wrong thing with two routes and two websockets. Does moving to one help performance?